src/stream/src/executor_v2/lookup/sides.rs

```diff
@@ -61,11 +61,16 @@ pub(crate) struct ArrangeJoinSide<S: StateStore> {
     /// The column descriptors of columns in arrangement
     pub col_descs: Vec<ColumnDesc>,
 
-    /// Order rules of the arrangement (used for lookup)
+    /// Order rules of the arrangement (only join key is needed, pk should not be included, used
+    /// for lookup)
     pub order_rules: Vec<OrderPair>,
```

Comment:
I saw a definition of `arrangement_order_rules` in `LookupExecutorParams` that mentioned the primary key of MV.  And the `order_rules` here is created from `arrangement_order_rules`. So what is the difference between those two order_rules?
```rust
    /// Should be the same as [`OrderPair`] in the arrangement.
    ///
    /// Still using the above `a, b, _row_id` example. If we create an arrangement with join key
    /// `a`, there will be two elements in `arrangement_order_rules`.
    ///
    /// * The first element is the order rule for `a`, which is the join key. Join keys should
    ///   always come first.
    /// * The second element is the original primary key for the MV.
    ///
    /// These two parts now compose the arrange keys for the arrangement.
    pub arrangement_order_rules: Vec<OrderPair>,
```

Reply:
`arrangement_order_rules` and `order_rules` are exactly the same.